docs/working_with_core/working_with_core_topic_1.rst

```diff
@@ -0,0 +1,17 @@
+==========================
+Working with core, topic 1
+==========================
+
+First test topic
+
+Working with core subtopic 1
+============================
```

Comment:
1. For RST to work, you only need to be consistent with headers within a document.  That said, it would be nice if we were consistent across this repo for the new docs we are introducing. We could use the recommendation from our [rst cheatsheet](https://github.com/edx/mdrst/blob/master/mdrst.rst#headers). Note: When we copy in files, you don't need to rework all the heading levels if they use a different pattern. 
2. Note: Your subtopic is using the same `=`, so this won't actually be treated as a subtopic. Again, using the cheatsheet recommendation will fix this.

Reply:
Since this is a stub, the header inconsistency will go away. That being said, I'll use the recs in the RST cheatsheet for "real" content.

Reply:
Thanks. Confirming that we will get consistency later on real docs.

1. Can you confirm that you are clear that in rST, using the same symbol (in this case `=`) will give the same header level (in this case header-1), whether it is used over and under, or just under.  The over usage is optional.
2. Rather than fix or remove the subtopics, does it make sense to just delete this file and all unused files? See:
```
/Users/rraposa/edx/other/edx-developer-docs/docs/developers_guide/index.rst: WARNING: document isn't included in any toctree
/Users/rraposa/edx/other/edx-developer-docs/docs/extending/extending_topic_1.rst: WARNING: document isn't included in any toctree
/Users/rraposa/edx/other/edx-developer-docs/docs/extending/extending_topic_2.rst: WARNING: document isn't included in any toctree
/Users/rraposa/edx/other/edx-developer-docs/docs/named_releases.rst: WARNING: document isn't included in any toctree
/Users/rraposa/edx/other/edx-developer-docs/docs/new_to_edx/what_to_read_next.rst: WARNING: document isn't included in any toctree
/Users/rraposa/edx/other/edx-developer-docs/docs/technical_reference/index.rst: WARNING: document isn't included in any toctree
/Users/rraposa/edx/other/edx-developer-docs/docs/working_with_core/working_with_core_topic_1.rst: WARNING: document isn't included in any toctree
/Users/rraposa/edx/other/edx-developer-docs/docs/working_with_core/working_with_core_topic_2.rst: WARNING: document isn't included in any toctree
```
3. Are any of these unused docs ones we want to keep?  If so, could we add them where we think they should live?
Thank you.